include/nuttx/mm/iob.h

```diff
@@ -31,6 +31,7 @@
 
 #include <stdint.h>
 #include <stdbool.h>
+#include <nuttx/spinlock.h>
```

Comment:
The header needs nothing from nuttx/spinlock, please move the include to the source files